src/Dependencies/PooledObjects/Microsoft.CodeAnalysis.PooledObjects.shproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Globals">
-    <ProjectGuid>c1930979-c824-496b-a630-70f5369a636f</ProjectGuid>
```

Comment:
It looks like this GUID is still being used in Roslyn.sln and CrossPlatform.sln. The fix might be to update the Compiler.sln version instead?